mne_bids/mne_bids.py

```diff
@@ -752,11 +752,20 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     # KIT data requires the marker file to be copied over too
     if 'mrk' in raw._init_kwargs:
         hpi = raw._init_kwargs['mrk']
-        _, marker_ext = _parse_ext(hpi)
-        marker_fname = make_bids_basename(
-            subject=subject_id, session=session_id, task=task, run=run,
-            acquisition=acquisition, suffix='markers%s' % marker_ext,
-            prefix=os.path.join(data_path, bids_raw_folder))
-        sh.copyfile(hpi, marker_fname)
+        acq_map = dict()
+        if isinstance(hpi, list):
+            if _get_mrk_meas_date(hpi[0]) > _get_mrk_meas_date(hpi[1]):
+                raise ValueError('Markers provided in incorrect order.')
+            _, marker_ext = _parse_ext(hpi[0])
+            acq_map = dict(zip(['pre', 'post'], hpi))
```

Comment:
this will still fail if you have two markers which are `pre` ...

Reply:
I'm not sure what you mean sorry...

Reply:
If there are two marker files, is it a given that one marker would be before the experiment and one after?

Reply:
I believe so

Reply:
According to our discussion, that would be the most common case, but it's not a MUST

Reply:
The way we do it we wouldnt ever have a case where it isnt true